internal/utils/moduleversionlabels.go

```diff
@@ -0,0 +1,40 @@
+package utils
```

Comment:
How about changing the file name to something a bit more generic and also moving `getDriverContainerNodeLabel` and `getDevicePluginNodeLabel` to the same file?